src/components/renders/bootstrap/tab-caption/tab-caption-render.tsx

```diff
@@ -54,6 +54,7 @@ export class TabCaptionRender implements Renderer {
 
   private clickHandler(event: UIEvent) {
     event.preventDefault();
+    this.component.selected = true;
```

Comment:
I'm worried that, by changing the property here and emitting the `tabSelect` event in the next line, the event will get fired twice when the tab page is clicked. Can you verify it please?

Reply:
Now we are emitting the `tabSelect` event in two different places. How can we fire it in only one place, so we can have only one source of truth to decide when to emit the event?
We already have a `@Watch` on the `selected` property, so maybe we could take advantage of that.

Reply:
🤔